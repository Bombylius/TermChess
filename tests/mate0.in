f2 f4 e7 e5 g2 g4 d8 h4 .q
