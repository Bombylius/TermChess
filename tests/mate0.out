35321811042304
269484032
70643622084608
550831656960
rnb1kbnr/pppp1ppp/8/4p3/5PPq/8/PPPPP2P/RNBQKBNR w KQkq - 1 3
1
